### PR TITLE
Fix Speed.CD parsing

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/torrent/html/speedcd.py [hunk 1]
```diff
@@ -152,12 +152,12 @@ def process_column_header(td):
...
                     download_url = urljoin(self.url,
-                                           cells[labels.index('Download')].find(title='Download').parent['href'])
+                                           cells[labels.index('Date')].find(title='Download').parent['href'])
                     if not all([title, download_url]):
                         continue
 
-                    seeders = try_int(cells[labels.index('Seeders')].get_text(strip=True))
-                    leechers = try_int(cells[labels.index('Leechers')].get_text(strip=True))
+                    seeders = try_int(cells[labels.index('Size')].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('Seeders')].get_text(strip=True))
```
Comment (fernandog) [inline medusa/providers/torrent/html/speedcd.py]: @duramato is this right? seeders = size?

**Comment by fernandog** [comment]
@duramato not working

**Comment by medariox** [comment]
I think this defeats the point of using index()...

File: medusa/providers/torrent/html/speedcd.py (hunk 1 above)
Comment (duramato) [inline medusa/providers/torrent/html/speedcd.py]: Yes

**Comment by duramato** [comment]
The index is all jumbled, it can be changed for the index value

**Comment by fernandog** [comment]
approved

**Pull request merged**